f2/src/utils/encryptedEmail.js

```diff
@@ -55,75 +57,75 @@ const getCert = uid => {
   })
 }
 
+const prepareUnencryptedReportEmail = (message, data, callback) => {
+  let transporter = nodemailer.createTransport({
+    streamTransport: true,
+    newline: 'unix',
+    buffer: true,
+  })
+
+  let attachments = data.evidence.files
+    .filter(file => !isProductionSystem || file.malwareIsClean)
```

Comment:
right now we're not connecting to Clam outside prod - just a hack so that we'll get files attached anyway.

Reply:
got our own clam and removed this hack.